src/styles/fonts.css

```diff
@@ -0,0 +1,11 @@
+@font-face {
+  font-family: Avenir;
+  src: url('../fonts/Avenir-Book.ttf');
+  font-weight: 400;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You can define a generic font family or multiple font families as a fallback. For example: font-family: Avenir, Arial, sans-serif;